loadflow/loadflow-api/src/main/java/com/powsybl/loadflow/json/LoadFlowResultSerializer.java

```diff
@@ -40,6 +40,7 @@ public void serialize(LoadFlowResult result, JsonGenerator jsonGenerator, Serial
         jsonGenerator.writeStringField("version", VERSION);
         jsonGenerator.writeBooleanField("isOK", result.isOk());
         jsonGenerator.writeObjectField("metrics", result.getMetrics());
+        jsonGenerator.writeObjectField("logs", result.getLogs());
```

Comment:
Logs can be very long, this is the reason why we never serialize them in JSON. I don't know if it's a good idea to serialize the logs as a long string.
@geofjamg Should we write the logs line by line, as an array object?